SimplenoteUITests/NoteEditor.swift

```diff
@@ -117,6 +117,14 @@ class NoteEditor {
     }
 
     class func getTextViewsWithExactValueCount(value: String) -> Int {
+        // We wait for at least one element with exact value to appear before counting
+        // all occurences. The downside is having one extra call before the actual count.
+        let predicate = NSPredicate(format: "value == '" + value + "'")
+        let _ = app
```

Comment:
Nitpick. Since we're ignoring the result, we can also omit the `let` definition.

```suggestion
        _ = app
```

Reply:
Thank you!